### PR TITLE
[NO-TICKET] Fix scan scripts after node 16 upgrade

### PR DESCRIPTION
**Pull request by pwolfert**
## Summary

On master, our `yarn scan-dependents` script was broken. This fixes it. I wasn't able to get our root tsconfig working with our scripts without messing it up for our ds packages, so I just made a new tsconfig.

### CODE REVIEW

File: scripts/tsconfig.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
+  // This is an alias to @tsconfig/node16: https://github.com/tsconfig/bases
+  "extends": "ts-node/node16/tsconfig.json"
```
Comment (forrestbaer) [inline scripts/tsconfig.json:3]: Interesting didn't know about this.

**Review by forrestbaer — APPROVED** [review]
Working great for me now with node 16

File: scripts/tsconfig.json (hunk 1 above)
Comment (pwolfert) [reply scripts/tsconfig.json:3]: Me neither. I didn't really want to have two separate configs, but it was taking too long to troubleshoot the other one

Comment (pwolfert) [reply scripts/tsconfig.json:3]: At least with this extends feature we're not actually _maintaining_ a separate set of configs

**Pull request merged**